docs/AnnotationSpec.md

```diff
@@ -4,24 +4,22 @@ For good user experience and reduce user effort, we need to design a good annota
 
 If users use NNI system, they only need to:
 
- 1. Use nni.get_next_parameter() to retrieve hyper parameters from Tuner, before using other annotation, use following annotation at the begining of trial code:
-    '''@nni.get_next_parameter()'''
+ 1. Annotate variables in code as:
```

Comment:
please give more explanation about usage, for example, there must be an assignment expression below it.